client/blocks/eligibility-warnings/index.jsx

```diff
@@ -117,29 +144,16 @@ const EligibilityWarnings = props => {
 
 EligibilityWarnings.propTypes = {
 	isEligible: PropTypes.bool.isRequired,
-	errors: PropTypes.arrayOf(
-		PropTypes.shape( {
-			title: PropTypes.string,
-			description: PropTypes.string,
-			actionUrl: PropTypes.string,
-		} )
-	),
-	pluginWarnings: PropTypes.arrayOf(
-		PropTypes.shape( {
-			title: PropTypes.string,
-			description: PropTypes.string,
-			actionUrl: PropTypes.string,
-		} )
-	),
-	widgetWarnings: PropTypes.arrayOf(
-		PropTypes.shape( {
-			title: PropTypes.string,
-			description: PropTypes.string,
-			actionUrl: PropTypes.string,
-		} )
-	),
 	backUrl: PropTypes.string,
-	translate: PropTypes.func,
+	translate: PropTypes.func
+};
+
+const mapStateToProps = state => {
+	const siteSlug = getSelectedSiteSlug( state );
```

Comment:
We also need to think about what happens when someone navigates to `All Sites` which makes the selected site null. Maybe this isn't a problem because of how the controller is setup. This is not my expertise.